src/rx5808-pro-diversity/settings.h

```diff
@@ -83,6 +89,13 @@ SOFTWARE.
     #define DIVERSITY_MAX_CHECKS 5
 #endif
 
+#ifdef FAST_DIVERSITY_SWITCHING
+    // see https://www.arduino.cc/en/Reference/PortManipulation
+    // PORTC means analog ports (PORTD is arduino 0-7, PORTB is 8-13)
+    // PC0 is the current value of A0 (receiver A) PC1 is value of A1 (receiver B)
+    #define TOGGLE_RECEIVER PORTC ^= (1 << PC0)|(1 << PC1);
```

Comment:
taking it further - we might use the arduino defined macros port_to_output and digital_pin_to_port to create some clever macros but thats kinda like recreating digitalWrite. maybe it's worth it -  we want to accomplish the instant switching of two inputs at the same time - it doesnt matter if it happens in 2 cycles or in 10 cycles as long as both are switched at the same time.